sql-server-2014-alwayson-existing-vnet-and-ad/azuredeploy.json

```diff
@@ -2,14 +2,12 @@
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
-
-    "newStorageAccountNamePrefix": {
+    "namePrefix": {
```

Comment:
Can we just generate this with uniqueString()?  We make things much more likely to succeed.  Note that uniqueString will be 13 chars so you'll have to use take() to grab the first 8.